ScheduleSetting/ScheduleTable/story.jsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import {
+  action,
+  storiesOf,
+} from '@kadira/storybook';
+import { checkA11y } from 'storybook-addon-a11y';
+import ScheduleTable from './index';
+
+const times = [
+  {
+    value: {
+      hours: 9,
+      minutes: 41,
+    },
+    onChange: action('on-change'),
+  },
+  {
+    value: {
+      hours: 19,
+      minutes: 0,
+    },
+    onChange: action('on-change'),
+  },
+];
+
+const timesSingle = [
+  {
+    value: {
+      hours: 9,
+      minutes: 41,
+    },
+    onChange: action('on-change'),
+  },
+];
+
+const timesEmpty = [];
+
+const days = [
```

Comment:
Awesome! Rewriting our story data in this way has removed around 130 lines of code! 😱